packages/web-demo/src/app/bootstrap/components/badge/badge-interactive-demo/badge-interactive-demo.component.html

```diff
@@ -0,0 +1,10 @@
+<a href="#" class="badge">Link Badge</a>
+<button type="button" class="badge">Button Badge</button>
+<div class="badge-check">
+  <input class="badge-check-input" type="checkbox" value="" id="CheckableBadge">
+  <label class="badge-check-label" for="CheckableBadge">Checkable Badge</label>
+</div>
```

Comment:
```suggestion
</div>
<div class="badge-check">
  <input class="badge-check-input" type="checkbox" value="" id="CheckableBadge">
  <label class="badge-check-label" for="CheckableBadge">
    Checkable Badge
    <span class="badge">10</span>
  </label>
</div>
```
Would be nice to have an example with a nested badge.